### DISCUSSION TITLE
반성합니다

### DISCUSSION THREAD

**Discussion by gusah009** [BDD-CLUB/challenge-every-week-posting#515] [6. 반성문]
아무나 저를 이 지옥에서 빼내주세요... 담주부턴 열심히 해보겠습니다!

![image](https://github.com/BDD-CLUB/challenge-every-week-posting/assets/26597702/b412e255-9bcd-46ae-b19e-70455d501dbb)

**Comment by jojaeng2** [comment]
별로 안하셨는데요?

**Reply by gusah009** [reply]
롤악귀조정현
